sourcecode/src-3/sourcecode/Macros.scala

```diff
@@ -189,8 +189,14 @@ object Macros {
         owner match {
           case defSym if defSym.isDefDef =>
             defSym.tree.asInstanceOf[DefDef].paramss
+              // FIXME Could be a List[TypeDef] too, although I'm not
+              // under which conditions this can happen…
+              .map(_.asInstanceOf[List[ValDef]])
```

Comment:
`paramss` is a `List[ParamClause[T]`, with `ParamClause[T]` being `List[ValDef[T]] | List[TypeDef[T]]`. So we can't just match `paramss` with
```scala
paramss match {
  case l: List[ValDef] =>
  case l: List[TypeDef] =>
}
```
because of type erasure.